### PR TITLE
Allow origin CHANNEL to be used in RemoteNickFormat

### PR DESCRIPTION
**Pull request by patcon**
Super simple. Will reference this from the commit to be cherry-picked in a sec.

### CODE REVIEW

**Comment by patcon** [comment]
Hopefully not an issue that I tried to reduce doc duplication too

File: matterbridge.toml.sample [hunk 1]
```diff
@@ -1298,6 +1245,7 @@ RemoteNickFormat="{NICK}"
 #The string "{BRIDGE}" (case sensitive) will be replaced by the sending bridge
 #The string "{LABEL}" (case sensitive) will be replaced by label= field of the sending bridge
 #The string "{PROTOCOL}" (case sensitive) will be replaced by the protocol used by the bridge
+#The string "{CHANNEL}" (case sensitive) will be replaced by the origin channel name used by the bridge
```
Comment (Helcaraxan) [inline matterbridge.toml.sample]: Had forgot to document my previous addition of `GATEWAY` in #501.

```suggestion
#The string "{GATEWAY}" (case sensitive) will be replaced by the origin gateway name that is 
replicating the message.
#The string "{CHANNEL}" (case sensitive) will be replaced by the origin channel name used by the bridge
```

**Comment by Helcaraxan** [comment]
Nice work on the deduplication!

**Comment by Helcaraxan** [comment]
Might actually be worth going the extra mile and solving #487 in the same commit.

**Comment by patcon** [comment]
@Helcaraxan heh, thanks. As for the other thing, it's not immediately clear to me how USERNAME / NICK differs across all bridge types, so I think I'll treat that as out-of-scope for me :)

**Comment by 42wim** [comment]
👍

**Pull request merged**